src/environment/environment.js

```diff
@@ -0,0 +1,127 @@
+/** @flow */
+import os from 'os';
+import { v4 } from 'uuid';
+import fs from 'fs-extra';
+import path from 'path';
+import npmInstall from '../utils/npm';
+import { loadScope } from '../scope';
+import { flattenDependencies } from '../scope/flatten-dependencies';
+import { BitId } from '../bit-id';
+import { Component } from '../consumer/component/consumer-component'
+import { BITS_DIRNAME } from '../constants';
+import { mergeAll } from 'ramda';
+
+const root = path.join(os.tmpdir(), 'bit');
+const currentPath = process.cwd();
+
+export default class Environment {
+  path: string;
+  component: Component;
+  componentsDependencies: ConsumerComponent[] = [];
+
+  constructor() {
+    this.path = path.join(root, v4());
+    console.log(this.path);
```

Comment:
It's helpful for now for debugging purposes, will be removed once this feature is finalized. (once we have a logging mechanism it'll be prettier).